tests/index.js

```diff
@@ -1,35 +1,224 @@
 const path = require('path');
 
+const rootTag = '.dc-page';
+
 module.exports = {
   'Open homepage': function (browser) {
     browser
       .url(browser.launchUrl)
-      .waitForElementVisible('body', 1000)
-      .end();
+      .useCss()
+      .waitForElementVisible(rootTag, 1000);
   },
   'Scroll to Anchor': function (browser) {
     const stepOptions = getStepOptions('scrollToAnchor', browser);
 
     if(!stepOptions) {
-      browser.end();
```

Comment:
@rbarilani I removed `end()`, to make tests run faster. As every `end` will close the browser and then next test will reopen it.
I added a test at last that closes the browser.